src/fsharp/FSharp.Core/Linq.fs

```diff
@@ -531,6 +532,7 @@ module LeafExpressionConverter =
             | LessQ      (_, _,[x1;x2]) -> transBinOp env false x1 x2 false Expression.LessThan           
             | LessEqQ    (_, _,[x1;x2]) -> transBinOp env false x1 x2 false Expression.LessThanOrEqual    
             | NotQ       (_, _, [x1])   -> Expression.Not(ConvExprToLinqInContext env x1)                                   |> asExpr
+            | LogicalNotQ (_, _, [x1])   -> Expression.Not(ConvExprToLinqInContext env x1)                                   |> asExpr
```

Comment:
Wow, `NotQ`, `LogicalNotQ`, and `BitwiseNotQ` all map to `Expression.Not`... That guy will pick if he represents logical or bitwise operation based on operand type and availability of the corresponding `op_*` guys: [source](https://github.com/dotnet/corefx/blob/690358247697a39f1c70dc1abe5bb08fa4680f84/src/System.Linq.Expressions/src/System/Linq/Expressions/UnaryExpression.cs#L612-L629).  The [docs](https://msdn.microsoft.com/en-us/library/system.linq.expressions.expression.not(v=vs.110).aspx) only mention the bitwise operation.

Sheesh, queries are confusing...